discord_bot/discord_bot.py

```diff
@@ -101,29 +127,33 @@ def get_gateway(self, **kwargs):
                                                                     self.config["gateway_encoding"])) as websocket:
             self.websocket = websocket
             while True:
-                response = await self.websocket.recv()
-                response = json.loads(response)
-                print("{}: {}".format(Opcodes(response["op"]).name, response))
-                if response["op"] == Opcodes.HELLO:
-                    asyncio.ensure_future(self.handshake(response))
-                elif response["op"] == Opcodes.HEARTBEAT_ACK:
+                message = await self.websocket.recv()
+                message = json.loads(message)
+                print("{}: {}".format(Opcodes(message["op"]).name, message))
+                if message["op"] == Opcodes.HELLO:
+                    asyncio.ensure_future(self.handshake(message))
+                elif message["op"] == Opcodes.HEARTBEAT_ACK:
                     pass
-                elif response["op"] == Opcodes.INVALID_SESSION:
+                elif message["op"] == Opcodes.INVALID_SESSION:
+                    self.last_seq = message["s"]
                     print('Invalid Session')
-                elif response["op"] == Opcodes.DISPATCH:
-                    self.last_seq = response["s"]
-                    event = response["t"]
-                    if event == 'READY':
-                        self.session_id = response["d"]["session_id"]
+                elif message["op"] == Opcodes.DISPATCH:
+                    self.last_seq = message["s"]
+                    event = message["t"]
+                    if event == EVENTS[1]:
```

Comment:
This is very unclear, the different events should be a separate `IntEnum` so it's readable what we're checking for here.